readme.md

```diff
@@ -139,7 +165,12 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+var Instructor = function Instructor(info) {
+  this.name = info.name,
+  this.receivePresent = function(){
+    this.gift;
+  }
+}
```

Comment:
You were getting close. Here's what it might look like:

```js
function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}

var andy = new Instructor("Andy");
andy.receivePresent("Resin Laying Deer Figurine, Gold");
```